.github/workflows/pr-docs-deploy.yml

```diff
@@ -51,14 +51,15 @@ jobs:
           body-include: '<!-- ELEMENT_PLUS_PREVIEW -->'
           number: ${{ steps.pr.outputs.id }}
 
-      - name: Comment playground link
+      - name: Comment playground link & docs preview
         if: ${{ success() }}
         uses: actions-cool/maintain-one-comment@v3.1.1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           body: |
             🧪 Playground Preview: https://element-plus.run/?pr=${{ steps.pr.outputs.id }}
-            Please comment the example via this playground if needed.
+            test Please comment the example via this playground if needed.
+            📄 Docs Preview: https://preview-${{ steps.pr.outputs.id }}-element-plus.surge.sh
```

Comment:
There is already a comment link about the document preview, so perhaps we don't need to add it here.

![image](https://github.com/user-attachments/assets/f7c4be9e-97c1-4281-ac58-d443ae72e0f5)

Reply:
I didn't know it was clickable. 😰

Reply:
Haha, it’s hard to realize until you try it. 🤣